src/Lykke.Service.MarketMakerReports.Core/Exceptions/InvalidPnLCalculationException.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Lykke.Service.MarketMakerReports.Core.Exceptions
+{
+    public class InvalidPnLCalculationException : Exception
```

Comment:
Please, add comment to describe when this exception can be thrown. Should this exception contains additional properties to analyze on top level?